var/spack/repos/builtin/packages/vesta/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+import os
+import fnmatch
+import shutil
+
+
+class Vesta(Package):
+    """VESTA is a 3D visualization program for structural models, volumetric data
+       such as electron/nuclear densities, and crystal morphologies."""
+
+    homepage = "http://jp-minerals.org/vesta"
+    url      = "https://jp-minerals.org/vesta/archives/3.4.6/VESTA-x86_64.tar.bz2"
+
+    version('3.4.6', '1d4651e86193f305831aa5db3dcfe789')
+
+    depends_on('gtkplus@2.1.0:')
+    depends_on('mesa')
+    depends_on('cairo@1.0:')
+    depends_on('gcc@5.4.0:')
+
+    def setup_environment(self, spack_env, run_env):
+        run_env.prepend_path('PATH', self.prefix)
```

Comment:
If you are doing this to generate module files with these prefixes, by default Spack will add `PATH`. You can also update `LD_LIBRARY_PATH` for all packages by updating `modules.yaml` (more on that at https://spack.readthedocs.io/en/latest/module_file_support.html#write-a-configuration-file)

Reply:
@scheibelp This won't work because Spack adds `prefix.bin` to `PATH` and `prefix.lib` to `LD_LIBRARY_PATH`. Unless we can install the right components to the right directories, we can't rely on Spack's automatic module support.